### DISCUSSION TITLE
wiliwili的配置目录

### DISCUSSION THREAD

**Discussion by xfangfang** [xfangfang/wiliwili#39] [Q&A]
PC 平台可以在应用内的 `设置-实用工具` 找到打开配置目录的菜单项直接点击打开配置目录

## switch
内存卡根目录：`/config/wiliwili`

## psv

ux0:/data/wiliwili

## macOS
`$HOME/Library/Application\ Support/wiliwili`

## Windows
`%HOMEPATH%\AppData\Local\xfangfang\wiliwili`

### - 普通安装方式
`$HOME/.config/wiliwili`

### - flatpak安装
`$HOME/.var/app/cn.xfangfang.wiliwili/config/wiliwili`

## debug模式
> 这一节是写给开发者的

若在编译时未指定release，也就是未指定：`-DCMAKE_BUILD_TYPE=Release`
那么配置文件目录为软件执行目录：`./config/wiliwili`

## 相关问题
[PC客户端缺失字体的解决方法](https://github.com/xfangfang/wiliwili/discussions/38)
[如何自定义字体](https://github.com/xfangfang/wiliwili/discussions/40)

**Comment by redomCL** [comment]
大佬能否再Windows添加让配置文件与软件同目录
